templates/typescript/src/index.ts

```diff
@@ -0,0 +1,22 @@
+import "fastestsmallesttextencoderdecoder-encodeinto/EncoderDecoderTogether.min.js";
+import { run } from "./lib";
+
+import { setup, runnable } from "@suborbital/runnable";
+
+declare global {
+  var TextEncoder: any;
+  var TextDecoder: any;
+}
+
+const decoder = new TextDecoder();
+
+export function run_e(payload: ArrayBuffer, ident: number) {
+  // Imports will be injected by the runtime
+  // @ts-ignore
+  setup(this.imports, ident);
```

Comment:
Why do the imports need to be passed every time run_e is called?

Reply:
That was done just to keep the library setup simple (instead of having separate setup functions for the imports and the ident). It's just doing one assignment, so no real performance hit. We can change that in a future version though.

Reply:
Ok, please file an issue for that change

Reply:
https://github.com/suborbital/reactr/issues/207